include/variables.h

```diff
@@ -15,6 +15,9 @@ extern s16* D_80151328;
 extern s16 D_8010CD10;
 extern s16 D_8010CD12;
 extern s32 D_801595A0;
+extern u8 gCloudyFlowerFieldsBg; // "fla_bg"
```

Comment:
Does it work to make these `u8*`? plain `u8`s is kind of odd.

Reply:
`u8*` causes type warnings to appear when giving `strcmp(_, &gCloudyFlowerFieldsBg)` since it means we're actually giving it `u8**`.

Reply:
Dropping the `&` and using `u8*` compiles, but doesn't match. Using `&` or a string literal changes up the instruction order somewhat.

Reply:
imo, the declaration for this should be: ``extern char gCloudyFlowerFieldsBg[];`` If that works, then I'd prefer that.

Reply:
`char` over `u8`?

Reply:
this won't be a thing soon anyway, since it'll be a string literal, so I wouldn't worry too much

Reply:
Updated the code to use `char*` for string pointers and `char[]` for those externs as @pixel-stuck suggested - it matches.

RE: string literal, it caused the loading of the string address to happen earlier, which matches. Without a literal or `&` a few instructions were swapped. With the `char[]` typing I can omit `&` and it continues to work fine, however :sparkles: